src/sass/layout/_site-footer.scss

```
@@ -1 +1,47 @@
-.site-footer {}
+.site-footer {
+  align-content: flex-end;
+  background-color: black;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;   
+  padding:{ 
+    
+    bottom: 10px;
+    top: 40px;
+  }
+  
+   @media (max-width: 618px) {
+        margin: 0 auto;
+        display: block;        
+     
+  }
+  
+
+.footer-hover:hover{
+    color: yellow;
+  }
+  
+.list-contact{
+  display: block; 
+  color: white; 
+  list-style-type: none;
+  padding: 0;
+  padding-left: 70px;  
+  
+}
+  
+
+a{
+  text-decoration: none;  
+  color: white; 
+
+}
+
+.a:visited { 
+  color: white;  
+    }
```

Consider combining this selector with your rules on lines 20-22:
```suggestion
a{
  color: white;
  text-decoration: none;
  &:visited {
    color: white;
  }
  &:hover, &:focus {
    color: yellow;
  }
}
```